mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -126,6 +127,9 @@
 
     private final MessageParser messageParser;
 
+    private int limitOfAnnotations;
```

Comment:
It cannot be final I think because it can be initial by 2 ways. One as default value and the other as custom value for testing.